tests/components/sma/__init__.py

```diff
@@ -103,9 +122,7 @@ async def init_integration(hass):
     )
     entry.add_to_hass(hass)
 
-    with patch(
-        "homeassistant.helpers.update_coordinator.DataUpdateCoordinator.async_config_entry_first_refresh"
-    ):
+    with patch("pysma.SMA.read"):
         await hass.config_entries.async_setup(entry.entry_id)
     await hass.async_block_till_done()
     return entry
```

Comment:
Please avoid patching the config flow function `validate_input` below.

Reply:
Removed the validate_input and changed it to seperate patches of `pysma.SMA` where needed. fcfa095